melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/pdl/dto/sed/DnummerRekvisisjonBostedsadresse.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.melosys.eessi.integration.pdl.dto.sed;
+
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class DnummerRekvisisjonBostedsadresse {
```

Comment:
Er `Dnummer` og så videre en god konvensjon å bruke som navngivning?
Blir det da nye klasser når det kommer støtte for NPID?
Det virker som man burde velge et mer generisk begrep

Reply:
DTO-navnet har jeg kopiert fra PDL. Har du forslag til et bedre navn?

Reply:
Her tenker jeg om vi kunne samarbeide med PDL for å få endret deres API siden vi er eneste konsument.

`IdentRekvisisjon...` er et forslag som er mer generisk

Reply:
Har sendt melding til PDL og hører med de.

Reply:
Endret.